packages/DraftEditor/src/LinkControl/LinkControl.tsx

```diff
@@ -0,0 +1,103 @@
+import Input from "@blaze-react/input";
+import Modal from "@blaze-react/modal";
+import { ContentState, EditorState, SelectionState } from "draft-js";
+import React, { useState } from "react";
+import StyleButton from "../StyleButton";
+
+import { IMMUTABLE, LINK } from "../constants";
+import { ILinkControlProps } from "../interfaces";
+
+const LinkControl = ({
+  editorState,
+  unSelectedText,
+  onToggle
+}: ILinkControlProps): JSX.Element => {
+  const [modalStatus, setModalStatus] = useState<boolean>(false);
+  const [url, setUrl] = useState<string>("");
+  const [selectedContent, setSelectedContent] = useState<SelectionState>();
+
+  const getSelection = (): void => {
+    const selection: SelectionState = editorState.getSelection();
```

Comment:
could you do destructuring here like: 
``` 
const {getStartOffset, getEndOffset}: SelectionState = editorState.getSelection();

const start: number = getStartOffset();
const end: number = getEndOffset();
```

Reply:
when I do the destructuring for the selection, it throws me an error